README.md

```diff
@@ -1702,6 +1702,7 @@ HTML Themes and UI Kits to create your applications faster!
   * [Buttons Generator](https://markodenic.com/tools/buttons-generator/) — 100+ buttons you can use in your project.
   * [WrapPixel](https://www.wrappixel.com/) — Download High Quality Free and Premium Admin dashboard template created with Angular, React, VueJs, NextJs and NuxtJs!
   * [Utils.fun](https://utils.fun/en) — All offline daily and development tools based on the browser's computing power, including watermark generation, screen recording, encoding and decoding, encryption and decryption and code formatting, are completely free and do not upload any data to the cloud for processing.
+  * [Free Hugo Themes](hugo-templates.netlify.app) — A very nice selection of beautiful well designed responsive hugo themes for Hugo static site generator.
```

Comment:
Is there any reason these are not listed on the main hugo website? This site just seems like a basic ad to drive traffic to staticmania and fabforms